### PR TITLE
[Docs]Add word cloud description

### PR DESCRIPTION
**Pull request by sunshinemingo**
Add some description about the word cloud.

### CODE REVIEW

File: docs/report.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ We anaylsis 621,377,878 records of GitHub logs, there are 39,759,065 active repo
 
 ## GitHub Word Cloud
 
+We select topics of top 10w active repositories, filtered the topics with a frequency of more than 25. Then a word cloud map is generated based on the data. The word cloud visually highlights the topics that appear frequently, so that we can quickly get the more popular topics.
```
Comment (frank-zsy) [inline docs/report.md:7]: This file should be updated by robot, do not modify this one.

Comment (sunshinemingo) [reply docs/report.md:7]: OK, I will fix it.

**Comment by sunshinemingo** [comment]
@frank-zsy could you review this PR please?

**Pull request merged**